24/10/22 (16:46:35) | ERROR: [Errno 2] No such file or directory: '/home/caleb/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/caleb/.config/bpytop/themes'
24/10/22 (17:19:39) | ERROR: [Errno 2] No such file or directory: '/home/caleb/.config/bpytop/themes'
Traceback (most recent call last):
  File "/usr/bin/bpytop", line 1357, in refresh
    for f in os.listdir(d):
FileNotFoundError: [Errno 2] No such file or directory: '/home/caleb/.config/bpytop/themes'
